Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.5.3 (unreleased)
 
 Language Features:
+ * Yul: Require switch cases to have the same type.
```

Comment:
I think this is a bugfix. It was always supposed to have the same type, just the type validation isn't implemented properly.

Reply:
Yeah, wasn't sure where to put it, I'll move it.

Reply:
Moved.